src/app/components/StartScreen.tsx

```diff
@@ -29,7 +28,7 @@ function StartScreen() {
   };
 
   return (
-    <div className="h-auto p-4 my-auto">
+    <div className="h-auto p-4 my-auto overflow-auto">
```

Comment:
```suggestion
    <div className="h-auto p-4 my-auto content scroll-container">
```

this just adds the correct styling

Reply:
fixed